src/ConvergeApi.php

```diff
@@ -54,23 +66,28 @@ class ConvergeApi
      * @param boolean $live True to use the Live server, false to use the Demo server
      * @return null
      **/
-    public function __construct($merchant_id, $user_id, $pin, $live = true)
+    public function __construct($merchant_id, $user_id, $pin, $live = true, $insecure = false)
     {
         $this->merchant_id = $merchant_id;
         $this->user_id = $user_id;
         $this->pin = $pin;
         $this->live = $live;
+        $this->insecure = $insecure;
+    }
+
+    public function setHandler($handler)
+    {
+        $this->handler = $handler;
     }
 
     /**
      * Send a HTTP request to the API
      *
      * @param string $api_method The API method to be called
-     * @param string $http_method The HTTP method to be used (GET, POST, PUT, DELETE, etc.)
      * @param array $data Any data to be sent to the API
      * @return string
      **/
-    private function sendRequest($api_method, $http_method = 'GET', $data = null)
```

Comment:
Why have you removed the HTTP Method parameter? What happens if there's an API method that requires an HTTP method other than POST?